### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/general/project-partners-partner/contact

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#49]
**Describe the request**  
Correct notation in german help text

**Propose what exactly should be changed**  
change Email Adresse in E-Mail-Adresse

**Original setting or content**   
*question help text ger*  
Bitte geben Sie den Namen und eine Email Adresse an.

**Proposed setting or content**   
*new question help text ger*  
Bitte geben Sie den Namen und eine E-Mail-Adresse an.

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (ger)

**Comment by jmfrenzel** [comment]
Fixed in upstream rdmo https://github.com/rdmorganiser/rdmo-catalog/commit/0bc223e01691fb82f59556576b3bb3e78abc159d via https://github.com/rdmorganiser/rdmo-catalog/pull/30
